lxd/apparmor/instance.go

```diff
@@ -198,6 +198,13 @@ func instanceProfile(sysOS *sys.OS, inst instance) (string, error) {
 			execPath = execPathFull
 		}
 
+		// Check that /dev/sev exists to pass it to the AppArmor profile.
```

Comment:
I don't think the test is warranted, apparmor won't mind us listing a non-existent path in the profile, so let's keep it simple and just make it a one line addition to the profile.